images/cli-artifacts/Dockerfile.rhel

```diff
@@ -4,11 +4,15 @@ FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.12 AS builder
 WORKDIR /go/src/github.com/openshift/oc
 COPY . .
 RUN yum install -y --setopt=skip_missing_names_on_install=False gpgme-devel libassuan-devel
-RUN make cross-build-darwin-amd64 cross-build-windows-amd64 --warn-undefined-variables
+RUN make cross-build --warn-undefined-variables
 
 FROM registry.svc.ci.openshift.org/ocp/4.2:cli
 COPY --from=builder /go/src/github.com/openshift/oc/_output/bin/darwin_amd64/oc /usr/share/openshift/mac/oc
 COPY --from=builder /go/src/github.com/openshift/oc/_output/bin/windows_amd64/oc.exe /usr/share/openshift/windows/oc.exe
```

Comment:
Now that we want to include the arch, can we add hardlinks or symlinks or some such to these with explicit arches?  We'd need to keep the old paths around (at least as links) until `oc adm release extract` and the console downloads deployment had been ported to use the new paths.  And personally, I'd be happier if we just stuck to GOOS and used `darwin` instead of `mac`.

Reply:
> I didn't change any of the existing paths, as I wasn't sure where else they may be used. Can we leave that for a separate commit?

I think we should leave *dropping* the existing paths to future work.  But openshift/console-operator#343 can be simplified if we add the consistent paths in this PR.  Something like:

```Dockerfile
COPY /go/src/github.com/openshift/oc/_output/bin/ /usr/share/openshift/
RUN mkdir /usr/share/openshift/mac /usr/share/openshift/windows && ln /usr/share/openshift/darwin_amd64/oc /usr/share/openshift/mac/oc && ln /usr/share/openshift/windows_amd64/oc.exe /usr/share/openshift/windows/oc.exe
```